journeyman/webdev/markdown.md

```diff
@@ -0,0 +1,26 @@
+# Markdown 
+
+## Intro
+
+Markdown is a text-to-html tool for web developers. It allows you to write information that is easy write, and easy to write than it converts it to an html structure for you.
+
+## Suggested Learning
+
+- Read the Intro and Syntax on [Markdown](https://daringfireball.net/projects/markdown/)
+- Watch [this video](https://www.youtube.com/watch?v=_gknWWa2OF0) to see more clearly how it is written and used.
+- View the different .md files in this Webdev repository
+
+## Requirements
+
+- Edit/add a README.md file in your Apprentice Project
+- Include: 
+    - A header
+    - A description of your project
+    - A link (Possibly to your websites domain if you have one)
+    - An image
+
```

Comment:
Add a line telling them to show their work to their mentor to have them give them the pass off.